doc/docusaurus/docusaurus.config.js

```diff
@@ -0,0 +1,105 @@
+// @ts-check
```

Comment:
There used to be a bunch of docs pulled from `analysis-options.yaml`, with rationale on why we did or did not enable some of the standard rules. Can we restore it?

Reply:
Ah, nevermind -- looks like I never moved much of the docs into Markdown templates.

We can pull the comments from analysis-options.yaml in a follow-up.

Reply:
i've restored all of the previous docs to the `/static/docs` directory, so we can select the ones that are useful

Reply:
Cool, thanks -- I'll give these a trim now.